.travis.yml

```diff
@@ -1,11 +1,11 @@
-language: rust
+anguage: rust
 rust:
   - nightly
 cache: cargo
 env:
   global:
-  - DOCKERHUB_ORG=gnosispm
-  - DOCKERHUB_PROJECT=safe-client-gateway
+    - DOCKERHUB_ORG=gnosispm
+    - DOCKERHUB_PROJECT=safe-client-gateway
 services:
   - docker
 script:
```

Comment:
Looking at the logs of https://travis-ci.com/github/gnosis/safe-client-gateway/builds/186205255 I cannot find the logs for `cargo build --verbose --all --locked` and `cargo test --verbose --all --locked` are they still executed?

Reply:
They appear to be executed now. I still don't get why we get 24% coverage reported when it also shows, 78% for `/src` . I am wondering if this could be an issue with the `deploy` step that runs the script with the line `RUN cargo build --release --locked` a couple more times from the `Dockerfile`